uui/components/overlays/Alert.module.scss

```diff
@@ -10,13 +10,23 @@
 }
 
 .alert-wrapper {
-    padding: 12px 18px 12px 12px;
     display: flex;
+    align-items: center;
     overflow: hidden;
     min-width: 170px;
     border-radius: var(--uui-alert-border-radius);
 }
 
+.size-48 {
+    padding: 12px 18px 9px 9px;
+    height: 48px;
```

Comment:
I need to replace 'height' with 'min-height' because it's causing a problem. When we write larger text, the height should increase, but it remains at 48, causing the style to distort.